### ISSUE TITLE
Basic system controls via the Screenly web interface

### ISSUE THREAD

**Issue by git-pi** [Screenly/screenly-ose#121]
I was unable to get into the SSH interface of Screenly today (for some reason), though I could still hit the web interface successfully and ping both interfaces (eth0 and wlan0) perfectly well..  Rather than hauling out a keyboard and monitor, it would have been nice to have a "Restart Screenly" and/or "Shut Down Screenly" buttons on the main interface page that would launch the appropriate Linux commands in the background.

As an aside, Screenly had stopped displaying after 18 hours or so, hence my need to restart it.  Don't know what version I'm currently on - see other message regarding that...  ;-)

**Comment by riccla0321** [comment]
+1 to the password protected web interface.

**Comment by xbmcnut** [comment]
+1. Reboot, password protected back-end an build number displayed in 'System Info' would be greatly appreciated.

**Comment by danbsfc** [comment]
This would be fantastic! Having buttons for Shutdown & Reboot, along with password protection would make it a virtually perfect piece of kit. Got 2 stations running Screenly OSE now and highly recommend it to anyone who asks.

**Comment by krzimmer** [comment]
Being able to power cycle the pi via web interface would be a very nice feature.  As far as web interface protection, I'm using iptables to allow access to only trusted hostnames.  This is a fine security mechanism for now, but I would still like basic username:password authenication.  I don't know how to implement this in an elegant way.  My current plan to add basic authentication is to install Apache.  Apache would at as an authenication proxy to the Screenly webserver.  I would then block direct access to the Screenly webserver.

**Comment by markdegrootnl** [comment]
+1

**Comment by plittlefield** [comment]
+1

**Comment by toussa** [comment]
+1

**Comment by vpetersson** [comment]
Blocked by #876.

**Comment by rusko124** [comment]
This issue was solved in [Celery](https://github.com/Screenly/screenly-ose/tree/celery) branch(PR #1011 ).
The subsequent discussion, I recommend to move here #1081

@vpetersson I think we can close this.
